desktop/analytics/partner.js

```diff
@@ -22,4 +22,20 @@
       partner_slug: $(e.currentTarget).data('partner-slug')
     })
   })
+
+  analyticsHooks.on('renderedPartner2ContactInfo', function (options) {
```

Comment:
Maybe the delegated event approach would work in this case? Something like
```javascript
$('.partner2-content').on('click', '#partner2-contact .email-gallery', function (e) {
  //...
})
```
Not sure if it's better tho.

I don't have strong opinion about the event names. I always feel the camelCase is hard to read but it's javascript. 🤷‍♂️